### DISCUSSION TITLE
Bootstrap : robustness of the model parameters

### DISCUSSION THREAD

**Discussion by Percival24** [nlmixr2/nlmixr2#289] [Q&A]
Hi!

After obtaining my final popPK model using SAEM, I used a non-parametric bootstrap to evaluate the robustness of the model parameters. I performed the bootstrap (2000 runs) with replacement from the data.

Sometimes, in papers, authors report the success rate of the bootstrap. This is the percentage of runs that did not crash.

My question is: Is it a problem if all 2000 runs succeed, meaning that none of them crash?

Here are the bootstrap estimates with the 95% confidence intervals:

CL, liters per day: 0.39 [0.2 - 0.5]
V, liters: 11.1 [7 - 16]

IIV_CL, %: 69.6 [32.7 - 109.5]
IIV_V, %: 86.2 [0.3 - 201.2]

Prop. error model: 15.2 [8 - 24]

Best regards

Percival

**Comment by billdenney** [comment]
It is perfectly fine if mome of your bootstrap fits crash. The crash usually happens due to some kind of model instability. It looks like your model is very stable based on the tight confidence intervals.

So, no surprise, no problem.

**Comment by mattfidler** [comment]
The crash also depends a bit on the modeling tool and the estimation method too.

**Comment by Percival24** [comment]
Alright, thanks a lot !
